mlflow/sklearn/__init__.py

```diff
@@ -455,6 +455,11 @@ def load_model(model_uri):
     )
 
 
+_last_mlflow_run_id = None
```

Comment:
Note: I use a global variable to track the latest mlflow run, instead of attach the run id into model/prediction result dataset, because i found the `numpy.ndarray` does not allow to attach new attributes.